tests/js/set-tests.js

```diff
@@ -154,37 +188,36 @@ module.exports = {
 
     //
     // test filtering on object properties
-    async testSetObjectFilter() {
+    testSetObjectFilter() {
```

Comment:
Why remove the `async` from the test functions?

Reply:
No need, `await` is not utilized inside the function.

Reply:
I don't see any async operations, so I believe it is not required.